src/apps/material/material.tsx

```diff
@@ -239,7 +239,11 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
         disclosureIconExpandAltText=""
         dataCy="material-details-disclosure"
       >
-        <MaterialDetailsList className="pl-80 pb-48" data={detailsListData} />
+        <MaterialDetailsList
+          id="main-details"
```

Comment:
Okay here I go talking about naming things again. 🙈 But maybe I would consider aligning these id's - one ID is defined here, and the other one inside the `MaterialManifestationItem` on line 102. In this case the main details component would have an id of `main-details` and the other manifestation details would be called something like `material-details-870970-basis:52643414` and so on. Would it make more sense if it was `material-details` for the main one & `material-details-870970-basis:52643414` using IDs as it is for the other ones?
It's food for thought - but I am approving this PR. Good job 🙂